src/steps/zsh.rs

```diff
@@ -239,7 +239,7 @@ pub fn run_oh_my_zsh(ctx: &ExecutionContext) -> Result<()> {
         println!("Pulling custom plugins and themes");
         ctx.git().multi_pull(&custom_repos, ctx)?;
     }
-
+    env::set_current_dir(&oh_my_zsh)?;
```

Comment:
Thanks for your interest in contributing to Topgrade!

Well, we do have an [issue](https://github.com/topgrade-rs/topgrade/issues/519) requesting this, but I am not sure this is needed, please see that issue for more information

Reply:
thanks for the feedback, indeed I tried to explicitly set `export ZSH=$HOME/.oh-my-zsh` instead of `ZSH=$HOME/.oh-my-zsh` in `~/.zshrc` and it fixed the problem. You will close this PR.

Reply:
Hi, I will close this PR considering that this PR is not something planned.